src/pages/index.tsx

```diff
@@ -100,17 +106,20 @@ const Index: React.FC<IndexProps> = ({ fetchedData }) => {
 
 export default Index;
 
-export const getServerSideProps: GetServerSideProps = withSSRLogger(
-  async () => {
-    const fetchedData = await getIdentities();
-
-    return {
-      props: {
-        fetchedData,
-      },
-    };
-  },
-);
+export const getServerSideProps: GetServerSideProps = async () => {
+  const sortedIdentities = await getIdentities().then((result) => {
+    if (result instanceof Error) {
+      throw new Error();
+    }
+    return sortIdentities(result);
+  });
+
+  return {
+    props: {
+      sortedIdentities,
+    },
+  };
+};
```

Comment:
This should be wrapped with `withSSRLogger`:

```ts
export const getServerSideProps: GetServerSideProps = withSSRLogger(
  async () => {
    const sortedIdentities = await getIdentities().then((result) => {
    if (result instanceof Error) {
      throw new Error(result);
    }

    return sortIdentities(result);
  });

  return {
    props: {
      sortedIdentities,
    },
  };
);
```

Also, you can throw the actual error (i.e. `throw new Error(result);`)

Reply:
OK done 629b6f1